gb-rtc/src/naive.rs

```diff
@@ -2,7 +2,7 @@ use crate::{
     constant::{DAY, HOUR, MAX_DAYS, MAX_TIME, MINUTE},
     ReadRtcRegisters, WriteRtcRegisters,
 };
-use std::time::Instant;
+use std::time::{Instant, SystemTime};
 
 #[derive(PartialEq, Eq, Debug)]
 pub struct Naive {
```

Comment:
make `Naive` implement the traits `Serialize + Deserialize`

Reply:
I can't since it contains the `Instant` type

Reply:
implement without using `derive`, like that you can use `NaiveSave`